terraform/environments/corporate-staff-rostering/locals_production.tf

```diff
@@ -150,6 +150,45 @@ locals {
         }
       }
 
+      pd-csr-a-8-b = {
+        config = merge(module.baseline_presets.ec2_instance.config.default, {
+          ami_name                      = "pd-csr-a-8-b"
+          ami_owner                     = "self"
+          availability_zone             = "${local.region}b"
+          ebs_volumes_copy_all_from_ami = false
+        })
+        instance = merge(module.baseline_presets.ec2_instance.instance.default, {
+          instance_type           = "m5.4xlarge"
+          disable_api_termination = true
+          disable_api_stop        = true
+          monitoring              = true
+          vpc_security_group_ids  = ["domain", "app", "jumpserver"]
+          tags = {
+            backup-plan         = "daily-and-weekly"
+            instance-scheduling = "skip-scheduling"
+          }
+        })
+        ebs_volumes = {
+          "/dev/sda1" = { type = "gp3", size = 128 } # root volume
+          "/dev/sdb"  = { type = "gp3", size = 128 }
+          "/dev/sdc"  = { type = "gp3", size = 128 }
+          "/dev/sdd"  = { type = "gp3", size = 112 }
+        }
+        tags = {
+          description       = "Migrated server PDCAW00008"
+          app-config-status = "pending"
+          csr-region        = "Region 2"
+          os-type           = "Windows"
+          ami               = "pd-csr-a-8-b"
+          component         = "app"
+        }
+        route53_records = {
+          create_internal_record = true
+          create_external_record = true
+        }
+      }
+
+
```

Comment:
Remove these blank lines, please. We only need one blank line between blocks.

Reply:
done